core/perf_test/PerfTest_Random.cpp

```diff
@@ -0,0 +1,105 @@
+//@HEADER
+// ************************************************************************
+//
+//                        Kokkos v. 4.0
+//       Copyright (2022) National Technology & Engineering
+//               Solutions of Sandia, LLC (NTESS).
+//
+// Under the terms of Contract DE-NA0003525 with NTESS,
+// the U.S. Government retains certain rights in this software.
+//
+// Part of Kokkos, under the Apache License v2.0 with LLVM Exceptions.
+// See https://kokkos.org/LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//@HEADER
+
+#include <Kokkos_Core.hpp>
+#include <Kokkos_Random.hpp>
+#include <benchmark/benchmark.h>
+#include "Benchmark_Context.hpp"
+
+namespace Benchmark {
+
+// Fills each entry of
+// * a view of size N
+// * with the sum of K random numbers
+// * between 0 and I
+template <typename Pool, typename Scalar>
+static void Random(benchmark::State &state) {
+  const size_t N = state.range(0);
+  const size_t K = state.range(1);
+  const size_t I = state.range(2);
+
+  Kokkos::View<Scalar *> out("out", N);
+  Pool random_pool(/*seed=*/12345);
+
+  for (auto _ : state) {
```

Comment:
Won't this create a warning for unused variable "_"?

Reply:
Actually it looks like some pretty recent clangs will still warn on this, let me fix it